<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>781</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graphs and Video</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 40, 49);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Chakra Petch Medium</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
}

PlotWidget {
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
}

QLabel, QCheckBox {
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
}

*{
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="PlotWidget" name="plotwidget_2" native="true">
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="PlotWidget" name="plotwidget_6" native="true">
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PlotWidget" name="plotwidget_1" native="true">
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="PlotWidget" name="plotwidget_3" native="true">
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="PlotWidget" name="plotwidget_4" native="true">
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="PlotWidget" name="plotwidget_5" native="true">
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="PlotWidget" name="plotwidget_7" native="true">
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="PlotWidget" name="plotwidget_8" native="true">
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="4,0,1,1,1">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="videolabel">
        <property name="text">
         <string>video</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="graphzoomenable">
        <property name="styleSheet">
         <string notr="true">border-radius: 15px</string>
        </property>
        <property name="text">
         <string>Enable Graph Zoom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopbutton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Chakra Petch Medium</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>stop video stream</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startbutton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Chakra Petch Medium</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>start video stream</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BatteryIndicator" name="batteryWidget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BatteryIndicator</class>
   <extends>QWidget</extends>
   <header>myWidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
